cpp/epidemiology/abm/world.cpp

```diff
@@ -82,6 +82,7 @@ void World::migration(TimePoint t, TimeSpan dt)
                 auto target_type = rule.first(*person, t, dt, m_migration_parameters);
                 Location* target      = find_location(target_type, *person);
                 if (target != & get_location(*person)) {
+                    target->get_testing_scheme().run_scheme(*person, m_testing_parameters);
```

Comment:
Yes, that makes sense.

It would be nice to combine the testing scheme with the migration rules, but one needs the location of the target location to run the testing scheme. I think, to get the location one needs the world in migration rules. If I remember correctly, we do not want to do this?

I added a check in `World::migration` such that a positive tested person cannot migrate. But I am open to find a nicer solution.